2025/pointer-events-wg.html

```diff
@@ -156,8 +156,9 @@ <h2>Scope</h2>
         <p>This Working Group seeks to enhance the features delivered in the <a href="https://www.w3.org/TR/pointerevents3/">Pointer Events Recommendation</a> by exploring changes, such as:</p>
           <ul>
             <li>Addition of <a href="https://www.w3.org/TR/pointerevents3/#details-of-touch-action-values">direction-specific values for <code>touch-action</code></a>, giving finer-grained control of panning touch behaviors</li>
-            <li>Clarifying relationship between pointer events and other behaviors/event models, such as mouse events, pointer lock, and drag and drop</li>
-			<li>Support for simplified basic high-level gestures for common interaction, such as "swipe left" or "two-finger tap"</li>
+			<li>Integrating mouse events and wheel events from the abandoned <a href="https://www.w3.org/TR/uievents/">UI Events</a> specification</li>
+            <li>Clarifying relationship between pointer events and other pointer-related APIs, such as pointer lock and drag and drop</li>
+			<li>Support for simplified basic high-level gestures, such as "swipe", "scale", "rotate"</li>
```

Comment:
Now that read this... "simplified basic high-level" is quite a mouthful. Would replacing "simplified basic" with "some" work here?

Reply:
or maybe even change it to "basic gestures"
```suggestion
			<li>Support for basic gestures, such as "swipe", "scale", "rotate"</li>
```